### ISSUE TITLE
[MEDIUM] audit: DASH - DASH model predice sistematicamente R/R < 1:1.5 (0.25, 0.42) ma i trade chiudono

### ISSUE THREAD

**Issue by fracarlesi** [fracarlesi/tradercripto#22]
## Summary
**Severity:** MEDIUM
**Model:** claude-sonnet-4-5

DASH model predice sistematicamente R/R < 1:1.5 (0.25, 0.42) ma i trade chiudono in profit grazie a exit anticipati non pianificati

## Bot state
- positions: 2
- equity: $50.14
- margin: $6.18
- leverage: 0.1x

## Flags
- **[MEDIUM] rr_prediction_asymmetric** (DASH): DASH model predicted tp=0.51% sl=2.00% (R/R=0.25, < 1:1.5)
- **[MEDIUM] rr_prediction_asymmetric** (DASH): DASH model predicted tp=0.83% sl=2.00% (R/R=0.42, < 1:1.5)

## Root cause hypotheses
- Il modello Qwen FLAG-Trader per DASH sottostima il potenziale di TP rispetto allo SL, possibile bias nel training set o feature engineering inadeguato per volatilità DASH
- Predicted SL fisso a 2.00% troppo conservativo per ATR ~0.9%, mentre TP predetti 0.51-0.83% sono troppo stretti, indicando calibrazione asimmetrica del modello
- I trade chiudono profittevoli (0.53%, 0.74%) prima di raggiungere TP predetto, suggerendo che exit logic reale (trailing stop o altro meccanismo non documentato) bypassa le predizioni del modello

## Suggested fix
1) Ricalibra il modello DASH: verifica loss function e target labels in training per bilanciare TP/SL ratio. 2) In config/model_overrides.yaml aggiungi constraint minimo R/R=1.5 per DASH o disabilita temporaneamente DASH trading. 3) Logga exit_reason_v2 nei prossimi trade per verificare se TP/SL predetti vengono effettivamente usati o se esiste logica di exit non documentata.

## Recent outcomes
```json
[
  {
    "timestamp": "2026-04-11T18:37:24.377483+00:00",
    "symbol": "DASH",
    "action": "SELL",
    "confidence": 1.8502072095870972,
    "entry_price": 46.753,
    "exit_price": 46.505,
    "pnl_usd": 0.072727,
    "pnl_pct": 0.530447244027116,
    "hold_duration_minutes": 4.45691505,
    "market_state_summary": {
      "rsi": 66.9,
      "adx": null,
      "regime": null,
      "atr_pct": 0.91,
      "ema9_slope": 1.104
    }
  },
  {
    "timestamp": "2026-04-11T18:52:35.487015+00:00",
    "symbol": "DASH",
    "action": "SELL",
    "confidence": 1.7468043565750122,
    "entry_price": 46.477,
    "exit_price": 46.135,
    "pnl_usd": 0.14525,
    "pnl_pct": 0.7358478387159213,
    "hold_duration_minutes": 8.033437133333333,
    "market_state_summary": {
      "rsi": 55.7,
      "adx": null,
      "regime": null,
      "atr_pct": 0.95,
      "ema9_slope": 0.404
    }
  }
]
```
